setup.py

```diff
@@ -30,51 +25,6 @@
     'Natural Language :: English',
 ]
 
-args = sys.argv[1:]
-
-# Make a `cleanall` rule to get rid of intermediate and library files
-if "cleanall" in args:
-    print("Deleting cython files...")
-    # Just in case the build directory was created by accident,
-    # note that shell=True should be OK here because the command is constant.
-    subprocess.Popen("rm -rf build", shell=True, executable="/bin/bash")
-    subprocess.Popen("find . -name \*.c -type f -delete", shell=True, executable="/bin/bash")
```

Comment:
The `setup.py` is much simpler thanks to `discretize` and the numpy config that @jcapriot has done!